jobrunner/sync.py

```diff
@@ -70,7 +70,10 @@ def api_request(method, path, *args, **kwargs):
     url = "{}/{}/".format(config.JOB_SERVER_ENDPOINT.rstrip("/"), path.strip("/"))
     # We could do this just once on import, but it makes changing the config in
     # tests more fiddly
```

Comment:
micronit: this comment is now false as we now rely on setting the headers per-request
```suggestion
```